packages/component-library/stories/index.js

```diff
@@ -57,11 +57,10 @@ sankeyStory();
 sliderStory();
 dropdownStory();
 baseMapStory();
-mapOverlayStory();
-hexOverlayStory();
 scatterPlotMapStory();
 pullQuoteStory();
 screenGridMapStory();
 pathMapStory();
 iconMapStory();
-boundaryMapStory();
```

Comment:
Why is `boundaryMapStory` being removed? The `BoundaryMap` component is still being exported in `src/index.js`.